The test fails also with the official PyPI package:
> torch/_inductor/select_algorithm.py:1869] [0/0] AssertionError: Input shapes should have M >= 16, N >= 16 and K >= 32
> ...
> torch._inductor.exc.InductorError: CompilationError: at 56:18:


Author: Alexander Grund (TU Dresden)

diff --git a/test/inductor/test_fp8.py b/test/inductor/test_fp8.py
--- a/test/inductor/test_fp8.py
+++ b/test/inductor/test_fp8.py
@@ -411,6 +411,7 @@ class TestFP8Types(TestCase):
         )
 
 
+@unittest.skip("Fails on H100s")
 @instantiate_parametrized_tests
 class TestFP8Lowering(TestCase):
     @unittest.skipIf(TEST_WITH_ROCM, "FP8 is not supported on ROCM")
a